docs/reference/api.yml

```diff
@@ -0,0 +1,3204 @@
+---
+openapi: 3.0.1
+info:
+  title: Open Food Facts Open API
+  description: As a developer, the Open Food Facts API allows you to get information and contribute to the products database. You can create great apps to help people make better food choices and also provide data to enhance the database.
+  termsOfService: https://openweathermap.org/terms
+  contact:
+    name: Open Food Facts
+    url: https://slack-ssl-openfoodfacts.herokuapp.com/
+    email: contact@openfoodfacts.org
+  license:
+    name: License (MIT, Apache 2.0, etc)
+    url: https://opendatacommons.org/licenses/odbl/summary/index.html
+  version: "2"
+servers:
+- url: https://world.openfoodfacts.org
+  description: prod
+paths:
+  /api/v2/product/{barcode}:
+    get:
+      tags:
+      - Read Requests
+      summary: Get for a specific product by barcode
+      parameters:
+      - name: barcode
+        in: path
+        description: The barcode of the product to be fetched
+        required: true
+        style: simple
+        explode: false
+        schema:
+          type: string
+          example: "04963406"
+      responses:
+        "200":
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/inline_response_200'
+components:
+  schemas:
+    inline_response_200:
+      type: object
+      properties:
+        code:
+          type: string
+        product:
+          type: object
+          properties:
+            _id:
+              type: string
+            _keywords:
+              type: array
+              items:
+                type: string
+            added_countries_tags:
+              type: array
+              items:
+                type: object
+            additives_debug_tags:
```

Comment:
There are quite a few of those response fields that don't exist anymore, or only on some old products.

Can we remove all the fields that containt "debug" or "old" from the documentation? We don't want anyone to use them.

Reply:
Fields with debug

additives_debug_tags, debug_param_sorted_langs, debug_tags, ingredients_debug, ingredients_ids_debug, ingredients_text_debug, nova_group_debugnutrition_score_debug, additives_debug_tags, debug_param_sorted_langs, debug_tags,  ingredients_debug, ingredients_ids_debug, ingredients_text_debug, nova_group_debug, nutrition_score_debug, 

Fields with old
additives_old_n, additives_old_tags , categories_old, labels_old, origins_old, packaging_old, packaging_old_before_taxonomization, additives_old_n, additives_old_tags, categories_old, labels_old, origins_old, packaging_old, packaging_old_before_taxonomization

@stephanegigandet before I make a push for this, I want to confirm that I am talking all these fields from the get_product_response schema. Removing them entirely.